src/main/java/align/GlobalOpt.java

```diff
@@ -171,25 +175,25 @@ public static void iterativeGlobalOpt(
 			{
 				int unaligned = tc.preAlign().size();
 				if ( unaligned > 0 )
-					System.out.println( "(" + new Date( System.currentTimeMillis() ) + "): pre-aligned all tiles but " + unaligned );
+					logger.info( "(" + new Date( System.currentTimeMillis() ) + "): pre-aligned all tiles but " + unaligned );
```

Comment:
Depending on the configuration, the logger might prepend a date string to log messages anyway. In this case, some logging messages can be simplified:  
```suggestion
					logger.info("pre-aligned all tiles except for {}", unaligned);
```

Reply:
Addressed by [2e4f4fe](https://github.com/PreibischLab/STIM/pull/8/commits/2e4f4fe5a0f440cbea312b37bee7deafb38f6814)